lib/bricks-net-http-mime_type.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+var GetFileExtension = require('./bricks-filesystem').GetFileExtension;
+
+
+var file_extension_to_mime_type_map = {
+    "js": "application/javascript",
+    "json": "application/json; charset=utf-8",
+    "css": "text/css",
+    "html": "text/html",
+    "htm": "text/html",
+    "txt": "text/plain",
+    "png": "image/png",
+    "jpg": "image/jpeg",
+    "jpeg": "image/jpeg",
+    "gif": "image/gif",
+    "svg": "image/svg+xml"
+};
+
+
+exports.GetFileMimeType = function () {
+	return cppArguments.assert('GetFileMimeType', [
+		[
+			cppArguments.assertion('string', 'const std::string&', 'pfile_nameath'),
```

Comment:
Fixed in https://github.com/sompylasar/KnowSheet-REPL/commit/72b082210a68480b224c47edb43757a94421c10c